README.md

```diff
@@ -1,37 +1,47 @@
 # kmt-utilities
-Common middleware for Knowledge Manager Tools (KMT) microservices
+Common middleware for KAT microservices.
 
-## Installation guide
+KAT (Knowledge & administration tools) is a part of ft.com application created for Financial Times B2B clients.
 
-Include ``"kmt-utilities": "financial-times/kmt-utilities"``as a dependency in your package.json
+## Getting started
+To get a copy of the project up and running on your local machine for development and testing purposes run `$ git clone git@github.com:Financial-Times/kmt-utilities.git`.
 
-Set up a utilities.Config.js to pass any of the process.env values you want to to the utilities middleware. It should look something like this:
+Use the `.env` file saved in LastPass to set up necessary environmental variables (feel free to approach KAT team via `#ft-syndikat` slack channel if you have any questions or access issues).
 
-    module.exports = {
-        API_GATEWAYKEY:  process.env.API_GATEWAYKEY,
-        SESSION_API_HOST:process.env.SESSION_API_HOST
-    };
+## Testing
 
-An API key for the services in this package is requried. Details on obtaining one for your application can be found in the [membership api gateway](https://developer.ft.com/docs/membership_platform_api/) guide.
+[TBD - when new tests are implemented]
 
-## Usage
+## Deployment
+This component has been created to be included throughout other KAT components.
```

Comment:
Just for the sake of consistency, maybe check whether "component" is the correct term, or whether it should be "module".